### PR TITLE
fix: remove HPA version from Data Overlay documentation

### LINKED ISSUE #738: Fix hard coded version of The Human Protein Atlas
**Describe the bug**
Currently the version for The Human Protein Atlas is hard coded to be v18, which is actually not what we use

**To reproduce**
Visit https://metabolicatlas.org/documentation and search for v18

**Expected behavior**
Either that we don't specify the version at all or parse it from data-files repo

**Screenshots**

**Others**
Please try to fix other potentially affected places.

### PR DESCRIPTION
**Pull request by inghylt**
**Related issue(s) and PR(s)**  
This PR closes #738

**Type of change**  

- [x] Other, improve documentation
 
**List of changes made**  

Data Overlay documentation is rewritten to not reference a specific HPA version as this easily can go out of date

Note that [this comment](https://github.com/MetabolicAtlas/MetabolicAtlas/issues/738#issuecomment-966336807) was fixed in #807

**Testing**  

Look at `/documentation#Data-overlay` and see if it looks good

**Further comments**  

**Checklist**  

- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by e0 — APPROVED** [review]
LGTM

File: frontend/src/components/Documentation.vue [hunk 1]
```diff
@@ -368,10 +364,10 @@
                 ). To clear the RNA levels, select the
                 <i>None</i>
                 option in the drop down list. RNA levels are available for both 2D and 3D Map
-                Viewer. The data is obtained from version 18 of the Protein Atlas with the units in
-                log
-                <sub>2</sub>
-                (TPM+1) associated with a gradient colorbar.
+                Viewer. The gene expression levels used are obtained from the file
```
Comment (MalinAhlberg) [inline frontend/src/components/Documentation.vue:367]: Maybe add that this file is from protein atlas?

**Review by MalinAhlberg — APPROVED** [review]
I left one comment/question, apart from that I think this looks good!

File: frontend/src/components/Documentation.vue [hunk 2]
```diff
@@ -368,10 +364,10 @@
...
+                <a href="https://www.proteinatlas.org/about/download" target="_blank">
+                  rna_tissue_hpa.tsv
+                </a>
```
Comment (nanjiangshu) [inline frontend/src/components/Documentation.vue:370]:
```suggestion
                </a>.
```

Comment (nanjiangshu) [reply frontend/src/components/Documentation.vue:370]: Sounds a bit weird when one click the url for the file `rna_tissue_hpa.tsv` but guided to a download page in which the file can not be found without searching. 
What about 
```html
The gene expression levels used are obtained from the file <b>rna_tissue_hpa.tsv.zip</b> in the <a href="https://www.proteinatlas.org/about/download" target="_blank">download page</a> of the Protein Atlas.
```
?

**Review by nanjiangshu — CHANGES_REQUESTED** [review]
Looks good @inghylt! Just some minor comments.

File: frontend/src/components/Documentation.vue (hunk 1 above)
Comment (inghylt) [reply frontend/src/components/Documentation.vue:367]: "For Human-GEM, the gene expression levels for genes from The Human Protein Atlas can be loaded" is two sentences before this one. I think it's ok to only mention it once, is it ok with you as well?

**Review by nanjiangshu — APPROVED** [review]
👍

File: frontend/src/components/Documentation.vue (hunk 2 above)
Comment (inghylt) [reply frontend/src/components/Documentation.vue:372]: Nice suggestion! Implemented in 379c2781151774ae4e19ed36b172cc8813199cc7

**Pull request merged**